scripts/deploy-mainnet.sh

```diff
@@ -23,8 +23,13 @@ CurveLpTokenUniv3Callee=$(dapp create CurveLpTokenUniv3Callee \
   0x9759A6Ac90977b93B58547b4A71c78317f391A28 \
   0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2)
 
+TUSDCurveCallee=$(dapp create TUSDCurveCallee \
+  0xEcd5e75AFb02eFa118AF914515D6521aaBd189F1 \
```

Comment:
Although this seems to be the correct TUSD Curve Metapool, I was unable to verify its authenticity as it was deployed by a [Metapool factory](0x0959158b6040D32d04c301A72CBFD6b39E21c9AE) that doesn’t match the [Curve documentation](https://curve.readthedocs.io/ref-addresses.html#metapool-factory)

Reply:
That factory seems to be an old one, found it in one of their contract:
https://github.com/curvefi/curve-factory/blob/master/contracts/Factory.vy#L112

You can see Curve presents the pool address here - https://curve.fi/contracts.
Look for "tusd".

Reply:
correct.